packages/lisk-transactions/src/utils/get_transaction_bytes.ts

```diff
@@ -178,7 +177,7 @@ const transactionTypeAssetGetBytesMap: {
 	7: getAssetDataForTransferOutOfDappTransaction,
 };
 
-export const getAssetBytes = (transaction: BaseTransaction): Buffer =>
+export const getAssetBytes = (transaction: TransactionJSON): Buffer =>
```

Comment:
I think this function should be removed from here and moved to each transaction type.

Reply:
This whole file, actually.

Reply:
`getAsset` functions in this file will be used in each type class as they are implemented. as for removing them, I believe we want to keep `util` functions as-is for client-side for the time being? @shuse2

Reply:
yes, to not to make breaking change now, we should keep it for now, but not use it in the BaseTransaction etc.
Maybe add a comment to deprecate?